src/main/java/frc/robot/OI.java

```diff
@@ -215,14 +216,14 @@ public static POVButton getUpBuddyBalanceButton() {
             return new POVButton(kJoystick, kUpBuddyBalanceButton.getButtonID()); // This button will move the lift to the balanced position if it was in the deployed position
         }
 
-        public static JoystickButton getYellowCargoButton() {
-            kYellowCargo.setButtonAction("Yellow Cargo Lights");
-            return new JoystickButton(kJoystick, kYellowCargo.getButtonID());
+        public static JoystickButton getConeSignalButton() {
```

Comment:
Why did we change these here?  This is an arm pull request. 
Let's try to keep pull requests focused.  Makes it easier to see what changed with the feature.